src/main/resources/com/google/api/codegen/java/settings.snip

```diff
@@ -499,9 +503,20 @@
 @end
 
 @private builderMethods(xsettingsClass)
+  private static ConnectionSettings.Builder s_getDefaultConnectionSettingsBuilder() {
```

Comment:
Why the s_ naming convention. Have we used that elsewhere? Does it stand for static?